src/api/xxAMIDOxx.xxSTACKSxx.Infrastructure/DependencyRegistration.cs

```diff
@@ -56,14 +53,23 @@ public static void ConfigureProductionDependencies(WebHostBuilderContext context
             services.AddTransient<IApplicationEventPublisher, DummyEventPublisher>();
 #endif
 
-            if (Environment.GetEnvironmentVariable("USE_MEMORY_STORAGE") == null)
-                services.AddTransient<IMenuRepository, MenuRepository>();
-            else
-                services.AddTransient<IMenuRepository, InMemoryMenuRepository>();
+#if (CosmosDb)
+            services.Configure<CosmosDbConfiguration>(context.Configuration.GetSection("CosmosDb"));
+            services.AddCosmosDB();
+            services.AddTransient<IMenuRepository, CosmosDbMenuRepository>();
+#elif (DynamoDb)
+            //services.Configure<DynamoDbConfiguration>(context.Configuration.GetSection("DynamoDb"));
+            //services.AddDynamoDB();
+            //services.AddTransient<IMenuRepository, DynamoDbMenuRepository>();
+#else
+            services.AddTransient<IMenuRepository, InMemoryMenuRepository>();
+#endif
 
             var healthChecks = services.AddHealthChecks();
+            healthChecks.AddCheck<CustomHealthCheck>("Sample"); //This is a sample health check, remove if not needed, more info: https://docs.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/monitor-app-health
+#if (DynamoDb)
```

Comment:
This doesn't look right 🤔

Reply:
Resolved. Used the correct argument now.